candies/cli/parsers/standard.py

```diff
@@ -0,0 +1,63 @@
+from argparse import ArgumentParser
+from typing import List, Any, Iterable, Callable, Dict
+import inspect
+
+from candies.cli.arg import Arg
+from candies.cli.parsers.parser import Parser
+
+
+class StandardParser(Parser):
+    """A parser that uses the standard `argparse` package.
+
+    Constructs an instance of `ArgumentParser` from the provided function.
```

Comment:
I feel like we need to mention here how the `ArgumentParser` instance will be used.

For example:
```py
"""
...

It constructs an instance of `argparse.ArgumentParser` from the provided function,
and uses it to parse the CLI arguments into key-value pairs.

...
```

Reply:
Added.